src/xrpld/app/misc/WamrVM.h

```diff
@@ -16,7 +16,8 @@
     OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 */
 //==============================================================================
-#pragma once
+#ifndef RIPPLE_APP_MISC_WAMRVM_H_INCLUDED
```

Comment:
Why?  `#pragma once`  is the right way, supported by all  compilers for more than 10 years. #ifdef is very deprecated technique.

Reply:
It's how rippled does things. IMO we should be uniform with what rippled does.

Reply:
Ripple started more than 10yrs ago. At that time I believe not all the compilers support #pragma. So it was necessary evil. But now using c80 standards and creating additional fields for errors (make a mistake in 1 letter in #define and you can get a lot of very hard to find silent errors). 
I am strongly against it usage.

Reply:
I don't disagree, just IMO if it's done it should be done in the rest of rippled too. IMO consistency is more important.

Reply:
It is standard approach in rippled. For example when we decided to move from the Hungarian notation, we just start using new notation, and trying fix Hungarian where it is possible without huge changes.

Reply:
That's fair.